field_records/ubna_2023.csv

```diff
@@ -31,13 +31,19 @@ Date and Time deployed (local) , Date and Time recovered (local) , AudioMoth # ,
 2023-06-28T13:22:00 , 2023-07-05T13:37:00 , D , 004 , Foliage , 47° 39' 18.750'' N , 122° 17' 26.940'' W , 48000 , Medium , None , None , 4.243 , 3.743 , CT , MB , CT , recover-20230705 , Panasonic Batteries Daytime 0:00-24:00 UTC
 2023-06-28T13:39:00 , 2023-07-05T13:53:00 , E , 012 , Central Pond , 47° 39' 30.480'' N , 122° 18' 18.240'' W , 192000 , Medium , None , None , 4.141 , 3.745 , CT , MB , CT , recover-20230705 , Ikea Batteries Nighttime 3:00-13:30 UTC
 2023-06-28T13:39:00 , 2023-07-05T13:53:00 , F , 009 , Central Pond , 47° 39' 30.480'' N , 122° 18' 18.240'' W , 48000 , Medium , None , None , 4.186 , 3.781 , CT , MB , CT , recover-20230705 , Panasonic Batteries Daytime 0:00-24:00 UTC
-2023-07-05T13:27:00 , (DATE-TIME) , H , 003 , Telephone Field , 47° 39' 30.480'' N , 122° 17' 18.240'' W , 48000 , Medium , None , None , 4.060 , (VOLTAGE) , CT , CT , MB , (RECOVER-DATE) , Panasonic Batteries Daytime 0:00-24:00 UTC Configured before deployment
-2023-07-05T13:27:00 , (DATE-TIME) , B , 011 , Telephone Field , 47° 39' 30.480'' N , 122° 17' 18.240'' W , 192000 , Medium , None , None , 4.128 , (VOLTAGE) , CT , CT , MB , (RECOVER-DATE) , Ikea Batteries Nighttime 3:00-13:30 UTC
-2023-07-05T13:42:00 , (DATE-TIME) , G , 010 , Foliage , 47° 39' 18.750'' N , 122° 17' 26.940'' W , 192000 , Medium , None , None , 4.049 , (VOLTAGE) , CT , MB , MB , (RECOVER-DATE) , Ikea Batteries Nighttime 3:00-13:30 UTC New AudioMoth switched out for C
-2023-07-05T13:42:00 , (DATE-TIME) , D , 001 , Foliage , 47° 39' 18.750'' N , 122° 17' 26.940'' W , 48000 , Medium , None , None , 4.041 , (VOLTAGE) , CT , MB , MB , (RECOVER-DATE) , Ikea Batteries Daytime 0:00-24:00 UTC
-2023-07-05T13:57:00 , (DATE-TIME) , E , 002 , Central Pond , 47° 39' 30.480'' N , 122° 18' 18.240'' W , 192000 , Medium , None , None , 4.053 , (VOLTAGE) , CT , CT , MB , (RECOVER-DATE) , Ikea Batteries Nighttime 3:00-13:30 UTC
-2023-07-05T13:57:00 , (DATE-TIME) , F , 005 , Central Pond , 47° 39' 30.480'' N , 122° 18' 18.240'' W , 48000 , Medium , None , None , 4.104 , (VOLTAGE) , CT , CT , MB , (RECOVER-DATE) , Panasonic Batteries Daytime 0:00-24:00 UTC
\ No newline at end of file
+2023-07-05T13:27:00 , 2023-07-11T15:35:00 , H , 003 , Telephone Field , 47° 39' 30.480'' N , 122° 17' 18.240'' W , 48000 , Medium , None , None , 4.060 , 3.721 , CT , CT , MB , recover-20230711 , Panasonic Batteries Daytime 0:00-24:00 UTC Configured before deployment
+2023-07-05T13:27:00 , 2023-07-11T15:35:00 , B , 011 , Telephone Field , 47° 39' 30.480'' N , 122° 17' 18.240'' W , 192000 , Medium , None , None , 4.128 , 3.784 , CT , CT , MB , recover-20230711 , Ikea Batteries Nighttime 3:00-13:30 UTC
+2023-07-05T13:42:00 , 2023-07-11T15:19:00 , G , 010 , Foliage , 47° 39' 18.750'' N , 122° 17' 26.940'' W , 192000 , Medium , None , None , 4.049 , 3.780 , CT , MB , MB , recover-20230711 , Ikea Batteries Nighttime 3:00-13:30 UTC New AudioMoth switched out for C
```

Comment:
```suggestion
2023-07-05T13:42:00 , 2023-07-11T15:19:00 , G , 010 , Foliage , 47° 39' 18.750'' N , 122° 17' 26.940'' W , 192000 , Medium , None , None , 4.049 , 3.780 , CT , MB , MB , recover-20230711 , Ikea Batteries Nighttime 3:09-13:11 UTC New AudioMoth switched out for C
```

With these new deployments, I think you might have forgot to update the recording periods? I checked `recover-20230711/UBNA_010` and the recording period is from 3:09-13:11 UTC